lib/cassandra.js

```diff
@@ -211,7 +211,7 @@ Cassandra.prototype.executeSQL = function(cql, params, options, callback) {
    return callback && callback(err, data ? data.rows : null);
   }
 
-  this.client.execute(cql, serialize(params), { prepare : true, readTimeout: 30000 }, myCallback);
+  this.client.execute(cql, serialize(params), { prepare : true, readTimeout: this.settings.readTimeout }, myCallback);
```

Comment:
Is there a default timeout configured for this value? What if it's not provided?